llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -3585,7 +3585,7 @@ multiclass VPseudoConversion<VReg RetClass,
                              int sew = 0,
                              int TargetConstraintType = 1> {
   defvar suffix = !if(sew, "_" # MInfo.MX # "_E" # sew, "_" # MInfo.MX);
-  let VLMul = MInfo.value in {
+  let VLMul = MInfo.value, SEW=sew in {
```

Comment:
Not related to this patch, but can you check if there are more misses of setting `SEW`?

Reply:
Yes I will check and can add a new PR.